civis_jupyter_notebooks/log_utils.py

```diff
@@ -15,13 +15,17 @@ def setup_stream_logging():
     logger.propagate = False
     # make sure to grab orig stderr since things seem to get redirected a bit
     # by the notebook and/or ipython...
-    ch = logging.StreamHandler(stream=sys.__stderr__)
+    error_handler = logging.StreamHandler(stream=sys.__stderr__)
+    info_handler = logging.StreamHandler(stream=sys.__stdout__)
     formatter = logging.Formatter(
         fmt='[%(levelname).1s %(asctime)s %(name)s] %(message)s',
         datefmt="%H:%M:%S")
-    ch.setLevel(logging.INFO)
-    ch.setFormatter(formatter)
-    logger.addHandler(ch)
+    error_handler.setLevel(logging.WARNING)
+    info_handler.setLevel(logging.INFO)
+    error_handler.setFormatter(formatter)
+    info_handler.setFormatter(formatter)
+    logger.addHandler(error_handler)
+    logger.addHandler(info_handler)
```

Comment:
I believe that that sets the log level for the default stream handler that is created with the logger (which would point to stderr), but I'm not sure why our INFO logs would not also be sent to stderr then, but it may be that the `propogate = False` setting is preventing more than one streamHandler from processing the logs.

I think I'm going to rewrite this setup to be a little clearer. I'm also going to test how warn/error logs are currently handled, to see if they are actually written to both streams or not.